squarelet/auth_helpers/urls.py

```diff
@@ -0,0 +1,25 @@
+# Django
+from django.urls import path
+
+# Third party
+from rest_auth.views import (
+    LoginView, LogoutView, PasswordChangeView,
+    PasswordResetView, PasswordResetConfirmView
+)
+
+# Local
+from . import views
```

Comment:
I've settled on preferring absolute imports to relative imports - I know the code base still has some relative imports but have been meaning to switch - am open to discussing the merits of each